data/presentations.yaml

```diff
@@ -1,3 +1,146 @@
+- title: "Community Introductions: Ed Warnicke at Network Service Mesh"
+  link: https://www.youtube.com/watch?v=fxtG1wuLSDw&feature=youtu.be&t=17m37s
+  youtube: fxtG1wuLSDw
+  presenters: Ed Warnicke
+  organizations: Network Service Mesh
+  venue: SPIFFE Community Day
+  category: case-study
+  date: 24-Apr-2020
+  language: English
+  description: A brief overview of how the open source project Network Service Mesh is integrating SPIFFE/SPIRE
+  weight: 100
+- title: "Community Introductions: Domingo Kiser at Frontdoor"
+  link: https://www.youtube.com/watch?v=fxtG1wuLSDw&feature=youtu.be&t=14m41s
+  youtube: fxtG1wuLSDw
+  presenters: Domingo Kiser
+  organizations: Frontdoor
+  venue: SPIFFE Community Day
+  category: case-study
+  date: 24-Apr-2020
+  language: English
+  description: A brief overview of how Frontdoor is using SPIRE
+  weight: 100
+- title: "Community Introductions: Zero Trust in Health Care"
```

Comment:
I'd suggest making these titles a little punchier. eg. "How Anthem is using SPIRE"